lib/web_ui/lib/src/engine/surface/scene_builder.dart

```diff
@@ -244,6 +244,7 @@ class SurfaceSceneBuilder implements ui.SceneBuilder {
     ui.Clip clipBehavior = ui.Clip.none,
     ui.PhysicalShapeEngineLayer oldLayer,
   }) {
+    assert(color != null, 'color must not be null');
```

Comment:
Does this exist in flutter implementation as well? There were prior breakages of apps due to framework being more permissive than web version.

Reply:
Yeah, a few lines below we do color.value so null throws already. I'm just making it a nicer error message.

Reply:
ack